src/api/api-add-categories.ts

```diff
@@ -1,14 +1,14 @@
 import axios from "axios";
 
-export interface categoriesObjModel {
+export interface CategoriesObjModel {
```

Comment:
It is more like `Category` or `ICategory`. No need to use `SomethingObjModel` cause it is decreases readibility.